Source/ACE/Entity/Player.cs

```diff
@@ -45,6 +45,11 @@ public sealed class Player : MutableWorldObject
         public bool IsOnline { get; private set; }
 
         /// <summary>
+        /// ObjectId of the currently selected Target (only players and creatures)
+        /// </summary>
+        public uint SelectedTarget { get; set; }

```

Comment:
It's not right, but the UpdateObject messages are doing more harm than good at this point and we'll be needing to address this soon to find the right sequence and way for handling this part